DATA_SOURCES.md

```diff
@@ -253,14 +253,11 @@ For many European countries, data is available from [ENTSO-E](https://transparen
   - Other: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedCapacityPerProductionUnit/show)
   - Per Region Renewable: [Terna](https://www.terna.it/en/electric-system/statistical-data-forecast/evolution-electricity-market)
   - Wind & Solar: [IRENA](http://resourceirena.irena.org/gateway/countrySearch/?countryCode=ITA)
-- India: 
+- India
   - solar: [MNRE.GOV.IN](https://www.mnre.gov.in/solar/current-status/)
   - wind: [MNRE.GOV.IN](https://www.mnre.gov.in/wind/current-status/)
   - biomass: [MNRE.GOV.IN](https://www.mnre.gov.in/bio-energy/current-status)
-  - other: [mercomindia](https://mercomindia.com/solar-indias-installed-power-capacity/)
-- India (Andhra Pradesh): [wikipedia.org](https://en.wikipedia.org/wiki/Power_sector_of_Andhra_Pradesh)
-- India (Punjab): [PUNJABSLDC](http://www.punjabsldc.org/realtimepbGen.aspx)
-- India (Chhattisgarh, Delhi, Gujarat, Karnataka, Punjab, Uttar Pradesh): [National Power Portal](https://npp.gov.in/dashBoard/cp-map-dashboard)
+  - other: [Ministry of Power India](https://cea.nic.in/%e0%a4%b8%e0%a4%82%e0%a4%b8%e0%a5%8d%e0%a4%a5%e0%a4%be%e0%a4%aa%e0%a4%bf%e0%a4%a4-%e0%a4%95%e0%a5%8d%e0%a4%b7%e0%a4%ae%e0%a4%a4%e0%a4%be-%e0%a4%b0%e0%a4%bf%e0%a4%aa%e0%a5%8b%e0%a4%b0%e0%a5%8d%e0%a4%9f/)
```

Comment:
I checked out the CEA website and it looks like they're using NPP data anyways:

<img width="932" alt="Screenshot 2021-03-30 at 10 29 45" src="https://user-images.githubusercontent.com/7591224/112958589-e9202880-9142-11eb-9c6f-71be5ff48dd8.png">

So maybe the best thing to do is instead of CEA have the [NPP dashboard as a source instead](url)?